### PR TITLE
[FEATURE] Added `rit set priority` command

### PR DESCRIPTION
**Pull request by marcoscostazup**
**- What I did**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: pkg/cmd/set_priority.go [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+			}
+		}
+
+		err = s.SetPriority(repo.Name, int(priority))
+		if err != nil {
+			return err
+		}
+
+		priorityString := strconv.Itoa(int(priority))
+		successMsg := fmt.Sprintf(newRepositoryPriority, repoName, priorityString)
```
Comment (kaduartur) [inline pkg/cmd/set_priority.go:76]: Use this message format `"Now %q repository has priority %v"`.
You don't need to [convert](https://github.com/ZupIT/ritchie-cli/pull/260/files#diff-616f189c5ccc020c6937c5a5513b1844R75) the int64 to string.
To learn more about the print format in Golang access this link [Printing doc](https://golang.org/pkg/fmt/#hdr-Printing)

File: pkg/formula/repo/priority_setter.go [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+		ritHome: ritHome,
+		file:    file,
+		dir:     dir,
+	}
+}
+
+func (sm SetPriorityManager) SetPriority(repoName string, priority int) error {
+	var repos formula.Repos
+	repoPath := path.Join(sm.ritHome, reposDirName, reposFileName)
+	if sm.file.Exists(repoPath) {
```
Comment (kaduartur) [inline pkg/formula/repo/priority_setter.go:29]: Here, if not exist return an error to add a repository.

File: pkg/cmd/set_priority.go [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+	prompt.InputList
+	prompt.InputInt
+	formula.RepositoryLister
+	formula.RepositoryPrioritySetter
+}
+
+func NewSetPriorityCmd(il prompt.InputList, ii prompt.InputInt, rl formula.RepositoryLister, rs formula.RepositoryPrioritySetter) *cobra.Command {
+	s := SetPriorityCmd{il, ii, rl, rs}
+	cmd := &cobra.Command{
+		Use:     "priority",
```
[the remark this answers is not in the document]
Comment (marcoscostazup) [reply pkg/cmd/set_priority.go:27]: Sure, it is going to be `rit set repo-priority` then? Nice to me

File: pkg/cmd/set_priority.go (hunk 1 above)
Comment (marcoscostazup) [reply pkg/cmd/set_priority.go:76]: Nice! Thanks

**Pull request merged**